.github/CONTRIBUTING.md

```diff
@@ -72,18 +80,30 @@ public:
         }
     }
 
-    /** Keep class method brackets on their own line, and always utilize const for methods and
+    /**
+     * Keep class method brackets on their own line, and always utilize const for methods and
      * variables when possible.
      *
      * For documenting functions, we loosely follow Doxygen conventions. You don't need to document
      * every single function, but important ones should optimally have at least a one-line comment
      * explaining what it does.
      *
-     * @param speaker        The person speaking
+     * This returns a translatable string, using gettext's _ function.
+     *
+     * @param speaker_id        The person speaking
      */
-    void exclaim(const std::string& speaker) const
+    t_string exclaim(const std::string& speaker_id) const
     {
-        std::cerr << speaker << ": They're taking the Hobbits to Isengard!" << std::endl;
+        if (how_far_to_destination_ < 100) {
+            if (speaker_id == "signboard") {
+                return _("Oldwood — enter at own risk");
+            } else {
+                // TRANSLATORS: The lake is the small underground one in S06 Temple in the Deep
+                return _("Hmm, someone has written underneath “People who push to master without making pull requests should be thrown in the lake.”");
```

Comment:
We don't require pull requests, and in any case I don't think the joke about throwing people in the lake works. I do like the example use of `TRANSLATORS:` and quotes, though.

Reply:
I agree, it was excessive. Will replace with a more in-universe example.